src/test/TestResult.java

```diff
@@ -1,12 +1,20 @@
 package test;
 
 import domain.Exercise;
+import java.util.Scanner;
 
 public class TestResult {
 
     public static void main(String[] args) {
+        
+        Scanner in = new Scanner(System.in);
+        
+        System.out.println("Введите слово");
+        
+        String enterString = in.nextLine();
+        
+        boolean result = Exercise.IsPalindrome(enterString);

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.